src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionIndex/MotionTransformIndex.cs

```diff
@@ -39,81 +39,81 @@ namespace Tizen.NUI.Scene3D
     /// orientation.TransformType = MotionTransformIndex.TransformTypes.Orientation;
     /// </code>
     /// </example>
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// <since_tizen> 11 </since_tizen>
     public class MotionTransformIndex : MotionIndex
     {
         /// <summary>
         /// The list of component types what this MotionIndex can control.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 11 </since_tizen>
         [global::System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1717:Only FlagsAttribute enums should have plural names")]
```

Comment:
this warning suppression could probably be dropped, it is plural name and it's not Flags

Reply:
Thank you. Now I remove this suppress message.